script/Ciel/Day 9/09_01_CIEL09_2_1.txt

```diff
@@ -529,7 +591,8 @@ A wholehearted shriek... probably to beg for his life... spills out from the gap
 [sha:cac0c745481f5864e9101169fd159ba0ab952e59]{
 -- Page 315, Offset 22954.
 -- たった一日の審問時間しかないんだもの。これぐらいはやらないと、このコたちに騙されて殺された人たちの無念は晴らせないでしょう？」
-I only have a single day to interrogate them. I have to at least do this much, or else the souls of the people they deceived and killed won't be able to rest in peace, right?"
+I've only got one day to interrogate them after all.
+ If I don't do at least this much, the souls of the people they deceived and killed won't be able to rest in peace, will they?"
```

Comment:
I'm fairly sure the 審問 here is from 異端審問, which is a word Nasu uses occasionally to invoke the sense of "punishing heretics (a.k.a. vampires)", rather than any actual interrogation. Kinda like how it is in western RPGs.
>I've only got a single day to perform their inquisition.

Maybe there's a verb form? I dunno.

Reply:
Verb form would be inquisite, which is archaic, but I guess so is the Church. Though, the Technically Correct : Reported Typo ratio probably isn't worth it.